index.bs

```diff
@@ -50,19 +51,51 @@ This provides similar functionality as earlier drafts of the
 
 ## Concepts ## {#concepts}
 
-A <dfn>entry</dfn> is either a [=file entry=] or a [=directory entry=].
+An <dfn>entry</dfn> is either a [=file entry=] or a [=directory entry=].
 
 Each [=/entry=] has an associated <dfn for=entry>name</dfn>.
 
 A <dfn lt="file|file entry">file entry</dfn> additionally consists of <dfn for="file entry">binary
 data</dfn> and a <dfn for="file entry">modification timestamp</dfn>.
 
 A <dfn lt="directory|directory entry">directory entry</dfn> additionally consists of a [=/set=] of
-<dfn for="directory entry">entries</dfn>. Each member is either a [=file=] or a [=directory=].
+<dfn for="directory entry">children</dfn>, which are themselves [=/entries=]. Each member is either a [=file=] or a [=directory=].
 
-Issue: TODO: Explain how entries map to files on disk (multiple entries can map to the same file or
+[=/Entries=] can (but don't have to) be backed by files on the systems native file system,
+as such it is possible for the [=binary data=], [=modification timestamp=],
+and [=children=] of entries to be modified by applications outside of this specification.
+Exactly how external changes are reflected in the data structures defined by this specification,
+as well as how changes made to the data structures defined here are reflected externally
+is left up to individual user-agent implementations.
```

Comment:
I see two possible meanings here.

1) The logical structures in this spec are mapped to filesystems in implementation-defined ways. I suppose this is reasonable. We could add an example of concepts map to POSIX filesystems. This covers every operating system I'm aware of -- Windows implements a subset of POSIX.

2) The exact sequence in which filesystem modifications reach the file system are implementation-dependent. This seems reasonable, with the caveat that we do have some guarantees for atomic writes -- no visible filesystem changes until time T1, all changes (file data, size) visible by time T2.

Reply:
I think I kind of mean both meanings, but it's definitely a bit clumsy as written. Not sure how to improve, although I suspect I'll be tweaking/modifying this text anyway as I try to actually better define individual algorithms. I expect we might end up with non-normative notes at the various algorithms explaining some of the things (like atomic writes)...